localsettings.example.py

```diff
@@ -99,6 +99,7 @@
 SERVER_EMAIL = 'commcarehq-noreply@example.com'  # the physical server emailing - differentiate if needed
 DEFAULT_FROM_EMAIL = 'commcarehq-noreply@example.com'
 SUPPORT_EMAIL = "commcarehq-support@example.com"
+DAILY_DEPLOY_EMAIL = "tech-announce-daily@example.com"
```

Comment:
We currently have all our other email addresses in `settings.py`, which gets automatically added to all the environments. I don't think that's necessarily the best idea, but I think good to follow convention for now so that everything is in one place for when that actually gets refactored.